flutter/test/file_system_transport_test.dart

```diff
@@ -0,0 +1,93 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry/sentry.dart';
+import 'package:sentry_flutter/file_system_transport.dart';
+
+void main() {
+  const MethodChannel _channel = MethodChannel('sentry_flutter');
+
+  const _jsonDecoder = JsonDecoder();
+  const _jsonEncoder = JsonEncoder();
```

Comment:
just curious : why not use `jsonEncode(...)` & `jsonDecode(...)`

Reply:
I didnt know about them, thanks :)